src/preamble.js

```diff
@@ -1701,34 +1701,6 @@ function addOnPostRun(cb) {
 }
 {{{ maybeExport('addOnPostRun') }}}
 
-// Tools
-
-/** @type {function(string, boolean=, number=)} */
-function intArrayFromString(stringy, dontAddNull, length) {
-  var len = length > 0 ? length : lengthBytesUTF8(stringy)+1;
-  var u8array = new Array(len);
-  var numBytesWritten = stringToUTF8Array(stringy, u8array, 0, u8array.length);
-  if (dontAddNull) u8array.length = numBytesWritten;
-  return u8array;
-}
-{{{ maybeExport('intArrayFromString') }}}
-
-function intArrayToString(array) {
-  var ret = [];
-  for (var i = 0; i < array.length; i++) {
-    var chr = array[i];
-    if (chr > 0xFF) {
-#if ASSERTIONS
-      assert(false, 'Character code ' + chr + ' (' + String.fromCharCode(chr) + ')  at offset ' + i + ' not in 0x00-0xFF.');
-#endif
-      chr &= 0xFF;
-    }
-    ret.push(String.fromCharCode(chr));
-  }
-  return ret.join('');
-}
-{{{ maybeExport('intArrayToString') }}}
-
```

Comment:
This block would be nice to be replaced by a `#include "arrayUtils.js"` block, instead of `src/jsifier.js` taking on the responsibility of concatenating that back in (see the comment above)